tests/meson.build

```diff
@@ -3,15 +3,24 @@ test_cases = [
   '../src/Lib/Util/FileSystem'
   ]
 
+# Tests fail to compile if the shared library is used on Windows
```

Comment:
@eli-schwartz Do you know why compiling the tests using the netpanzer shared lib fails on Windows but not on the other platforms we're testing on? (macos{amd64,arm64}, Ubuntu, and FreeBSD)

Reply:
Somehow libunwind is being linked in twice?

Reply:
@eli-schwartz Probably if you look at the improvements I just made in https://github.com/netpanzer/netpanzer/pull/183/commits/ebd7b4b029b3122a31d95bbff0ee7a44d7ccd3c3 you'll see the likely problem. Can this be improved anymore? Do I really need to declare two dependency objects, one for static and one for shared? Is there a bit of a shortcut?